src/Symfony/Component/Notifier/Bridge/Octopush/README.md

```diff
@@ -0,0 +1,25 @@
+Octopush Notifier
+===============
+
+Provides [Octopush](https://www.octopush.com) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+OCTOPUSH_DSN=octopush://USERLOGIN:APIKEY@default?from=FROM&sms_type=SMSTYPE
+```
+
+where:
+- `USERLOGIN` is your Octopush email
+- `APIKEY` is your Octopush token
+- `FROM` is your sender
+- `SMSTYPE` is Octopush sms type (XXX = SMS LowCost; FR = SMS Premium; WWW = SMS Monde)
```

Comment:
is it really `XXX`? or just a placeholder for anything?

Reply:
The expected parameter is the value 'XXX', yes. This is not an error.